make/hotspot/lib/CompileJvm.gmk

```diff
@@ -161,12 +161,17 @@ ifeq ($(call isTargetOs, linux), true)
   JVM_LDFLAGS += -Wl,-version-script=$(HOTSPOT_VERSION_SCRIPT)
 endif
 
+ifeq ($(call isTargetOs, ios), true)
+    LIB_TYPE := STATIC_LIBRARY
```

Comment:
Going forward, this will not be needed; running the proper make target will take care of making it static automatically.